### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by ArthiSiva1** [medusajs/medusa#10521]
### Package.json file

```JSON
{
  "name": "test",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "yarn eslint",
    "lint:fix": "yarn eslint --fix",
    "test": "jest --maxWorkers=4",
    "test:watch": "jest --watch",
    "test:coverage": "jest --coverage",
    "prepare": "husky"
  },
  "dependencies": {
    "@emotion/react": "^11.11.4",
    "@emotion/styled": "^11.11.5",
    "@medusajs/event-bus-redis": "^1.8.14",
    "@medusajs/js-sdk": "preview",
    "@medusajs/medusa-js": "^6.1.8",
    "@mui/icons-material": "^5.16.4",
    "@mui/lab": "^5.0.0-alpha.173",
    "@mui/material": "^5.16.1",
    "@mui/x-date-pickers": "^6.20.2",
    "@next/third-parties": "^14.2.14",
    "@strapi/blocks-react-renderer": "^1.0.1",
    "dayjs": "^1.11.13",
    "dompurify": "^3.1.6",
    "html-react-parser": "^5.1.16",
    "jwt-decode": "^4.0.0",
    "lodash": "^4.17.21",
    "next": "14.2.5",
    "react": "^18",
    "react-dom": "^18",
    "react-google-recaptcha": "^3.1.0",
    "react-hook-form": "^7.52.2",
    "schema-dts": "^1.1.2",
    "swiper": "^11.1.10",
    "tailwind-merge": "^2.5.2"
  },
  "devDependencies": {
    "@eslint/js": "^9.6.0",
    "@medusajs/medusa": "^1.20.7",
    "@next/eslint-plugin-next": "^14.2.5",
    "@stylistic/eslint-plugin": "^2.3.0",
    "@testing-library/dom": "^10.3.1",
    "@testing-library/jest-dom": "^6.4.6",
    "@testing-library/react": "^16.0.0",
    "@testing-library/react-hooks": "^8.0.1",
    "@testing-library/user-event": "^14.5.2",
    "@types/dompurify": "^3.0.5",
    "@types/jest": "^29.5.12",
    "@types/lodash": "^4.17.7",
    "@types/node": "^20",
    "@types/react": "^18.3.3",
    "@types/react-dom": "^18.3.0",
    "@types/react-google-recaptcha": "^2.1.9",
    "@types/testing-library__jest-dom": "^6.0.0",
    "autoprefixer": "^10.4.19",
    "eslint": "^9.8.0",
    "eslint-plugin-jest-dom": "^5.4.0",
    "eslint-plugin-react": "^7.34.3",
    "eslint-plugin-tailwindcss": "^3.17.4",
    "globals": "^15.8.0",
    "husky": "^9.0.11",
    "jest": "^29.7.0",
    "jest-environment-jsdom": "^29.7.0",
    "jest-sonar": "^0.2.16",
    "jest-sonar-reporter": "^2.0.0",
    "postcss": "^8",
    "react-password-strength-bar": "^0.4.1",
    "tailwindcss": "^3.4.1",
    "ts-jest": "^29.2.2",
    "ts-node": "^10.9.2",
    "typescript": "^5",
    "typescript-eslint": "^7.16.0"
  }
}
```

### Node.js version

20

### Database and its version

PostgreSQL 16

### Operating system name and version

Mac

### Browser name

Chrome

### What happended?

While performing login the user name is case sensitive. Login user name should not be case sensitive

### Expected behavior

User name should be case in sensitive

### Actual behavior

While performing login the user name is case sensitive. Login user name should not be case sensitive

### Link to reproduction repo

NA

**Comment by kasperkristensen** [comment]
Hi @ArthiSiva1, thanks for the report, but this issue has already been reported here: https://github.com/medusajs/medusa/issues/9886.

**Comment by rRamC** [comment]
@kasperkristensen I can see github-actions automatically closed #9886 

Please let us know when this bug is planned to be picked up.
